### DISCUSSION TITLE
ERROR: invalid tag " [ tag] : latest":  Invalid reference format

### DISCUSSION THREAD

**Discussion by Timmysworld** [docker/setup-buildx-action#291] [Q&A] [answered]
Hey, 
this is my first time messing with docker and the Github actions. I am following this link here https://docs.docker.com/build/ci/github-actions/ step by step. but whenever I try to run to see if it gets push to the docker hub i get this error: 

 [build](https://github.com/Timmysworld/.DotNetCICD/actions/runs/7467805226/job/20322018563#step:6:115)
buildx failed with: ERROR: invalid tag "***/.DotNetCICD:latest": invalid reference format 

not sure where to start to debug or why its doing that. Chat gpt share one thing but im not sure if i trust it being i am following a tutorial from the docker site.

**Comment by thaJeztah** [comment, marked as answer]
`.DotNetCICD` looks like it starts with a dot (.) which is not allowed in repository (image) names; likely should also be all lowercase

**Reply by thaJeztah** [reply]
Also see;

- https://docs.docker.com/engine/reference/commandline/tag/
- and (the technical implementation of image references) https://pkg.go.dev/github.com/distribution/reference

**Comment by Timmysworld** [comment]
Thank you that fixed my problem

**Reply by thaJeztah** [reply]
You're welcome!

**Reply by Aamir017** [reply]
buildx failed with: ERROR: invalid tag "/subzero:latest": invalid reference format 
facing the same issue

**Reply by thaJeztah** [reply]
@Aamir017 looks like that reference is missing something at the start; it cannot start with a `/` - is the registry prefix or username missing? (e.g. `yourname/subzero:latest` if it's on Docker Hub, or `some-registry.com/yourname/subzero:latest`)?

**Reply by QingminChen** [reply]
You save my life! It fixed my careless issue, thank you!
